webanno-ui-annotation/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/annotation/PreferencesUtil.java

```diff
@@ -114,6 +125,14 @@ public static void loadPreferences(String aUsername, SettingsService aSettingsSe
                         .getProject());
                 aBModel.setAnnotationLayers(layers);
             }
+
+            // Get color preferences for each layer, init with legacy if not found
```

Comment:
`Map<Long, LayerPreferences>` seems reasonable, I'd prefer that in a separate issue though

> As I said in ...

Yess, and now I also know much better why you mentioned that. I couldn't really follow before ;)